src/commonTest/kotlin/fr/acinq/lightning/wire/InitTlvTestsCommon.kt

```diff
@@ -1,16 +1,22 @@
 package fr.acinq.lightning.wire
 
-import fr.acinq.bitcoin.PublicKey
+import fr.acinq.bitcoin.Crypto
+import fr.acinq.bitcoin.DeterministicWallet
+import fr.acinq.bitcoin.MnemonicCode
 import fr.acinq.lightning.tests.utils.LightningTestSuite
 import fr.acinq.secp256k1.Hex
 import kotlin.test.Test
 import kotlin.test.assertEquals
+import kotlin.test.assertTrue
 
 class InitTlvTestsCommon : LightningTestSuite() {
     @Test
     fun `legacy phoenix TLV`() {
         val testCases = listOf(
-            Pair(PublicKey(Hex.decode("03864ef025fde8fb587d989186ce6a4a186895ee44a926bfc370e2c366597a3f8f")), Hex.decode("fe47020001 21 03864ef025fde8fb587d989186ce6a4a186895ee44a926bfc370e2c366597a3f8f"))
+            Pair(
+                first = DeterministicWallet.generate(MnemonicCode.toSeed("ghost runway obscure", passphrase = "")).publicKey,
```

Comment:
Why not use an actual real-life Phoenix seed?

Reply:
Done in 6362ac4.